Wallet Download and Installation.md

```diff
@@ -0,0 +1,64 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page from the Skycoin Website to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## [Desktop Wallet](Desktop_Wallet)
+
+* ### [Windows](Windows)
+* ### [MAC OS](MAC_OS)
+* ### [Linux](Linux)
+
+## [Mobile Wallet](Mobile_Wallet)
+
+* ### [IOS](IOS)
```

Comment:
I think this is not the right way to capitalize iOS. Please refer to Apple and update if necessary.